src/docfx/lib/github/GitHubAccessor.cs

```diff
@@ -46,18 +52,23 @@ public static async Task<UserProfile> GetUserProfileByName(string name)
             return ToUserProfile(user);
         }
 
-        private static UserProfile ToUserProfile(User user)
+        private UserProfile ToUserProfile(User user)
         {
             if (user == null)
                 return null;
 
-            // Can't get emails for now. Will try getting if after passing the token.
             return new UserProfile()
             {
                 ProfileUrl = user.HtmlUrl,
                 DisplayName = user.Name,
                 Name = user.Login,
                 Id = user.Id.ToString(),
+
+                // EmailAddress (public) can only be obtained with OAuth token (not required to be current user's token)
+                EmailAddress = user.Email,
+
+                // UserEmails can only be obtained with current user's OAuth token, so we only get the user's public email
+                UserEmails = user.Email,
```

Comment:
>UserEmails [](start = 16, length = 10)

this is emails, not email?

Reply:
This filed is intended to be email**s**, but usually the passed token can only obtain other user's one public email. Private emails can also be filled into this field from commit info.